cashctrl_ledger/ledger.py

```diff
@@ -77,11 +352,34 @@ def delete_price():
         """
         raise NotImplementedError
 
-    def delete_vat_code():
-        """
-        Not implemented yet
+    def delete_vat_code(self, code: str):
         """
-        raise NotImplementedError
+        Deletes a VAT code from the tax system via a POST request to the API.
+
+        Sends a POST request to the 'tax/delete.json' endpoint to delete the VAT code
+        specified by the 'code' parameter. The 'code' should be a string identifier
+        for the VAT code.
+
+        Parameters:
+        ----------
+        code : str
+            The identifier for the VAT code to be deleted.
+
+        Raises:
+        ------
+        Exception
+            An exception is raised with a detailed error message if the API request fails.
+
+        Returns:
+        -------
+        None
+            This method does not return any value but will raise an exception if unsuccessful.
+        """
+        try:
+            self.client.post('tax/delete.json', {'ids': code})
```

Comment:
`code` maps to the string identifier (CashCtrl 'name'), we need to map it to the 'id'.